### ISSUE TITLE
Why model uses BN layer

### ISSUE THREAD

**Issue by liudadan** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#295]
I did an experiment and found that the BN layer will have different outputs when inputting the same data in the train and eval states. Will using BN not destroy the characteristics of the data? Why does the author use the BN layer? Just to speed up your training?

**Comment by Daniil-Osokin** [comment]
Hi! It is a [standard behavior](https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm2d.html#batchnorm2d) for BN layer. In general yes, using BN often speedups the training in times.

**Comment by Daniil-Osokin** [comment]
Hope, it is clear now.
